### PR TITLE
Refactor/pro 799/distinguishable device ids

### PR DESCRIPTION
**Pull request by rdanitz**

### CODE REVIEW

**Comment by rdanitz** [comment]
end to end tests are supposed to fail due to change of serialization format

**Comment by rdanitz** [comment]
depends on advancedtelematic/ota-plus-test-driver#112

File: device-registry/src/test/scala/org/genivi/sota/device_registry/DeviceResourceSpec.scala [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright: Copyright (C) 2016, ATS Advanced Telematic Systems GmbH
+ * Copyright: Copyright (C) 2015, Jaguar Land Rover
```
Comment (CalumMcCall) [inline device-registry/src/test/scala/org/genivi/sota/device_registry/DeviceResourceSpec.scala]: I think this file was created in 2016.

**Comment by CalumMcCall** [comment]
Once the tests pass and the comment is addressed, this is good to merge.

**Pull request merged**